src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const isPersonParried = `${
```

Comment:
Use destructuring for `person`
```suggestion
export const Person = ({ person }) => {
  const { age, .......... } = person;

  const isPersonParried = `${
```